lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule_facts.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_sqlfirewallrule_facts
+version_added: "2.8"
+short_description: Get SQL Firewall Rule facts.
+description:
+    - Get facts of SQL Firewall Rule.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
+        required: True
+    server_name:
+        description:
+            - The name of the server.
+        required: True
+    name:
```

Comment:
not support tag?

Reply:
no, firewall rule is subresource that doesn't have own tags